sys/vistaff

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+from argparse import ArgumentParser
+from fcntl import flock
+from fcntl import LOCK_EX
+from fcntl import LOCK_NB
+from fcntl import LOCK_UN
+
+import yaml
+from ocflib.misc.shell import edit_file
+from staff_hours import get_staff_hours
```

Comment:
This should be `ocflib.lab.staff_hours` right? Obviously only after the ocflib change is made.